Libraries/src/Amazon.Lambda.Annotations.SourceGenerator/Models/AnnotationReport.cs

```diff
@@ -4,8 +4,25 @@ namespace Amazon.Lambda.Annotations.SourceGenerator.Models
 {
     public class AnnotationReport
     {
+        /// <summary>
+        /// Collection of annotated Lambda functions detected in the project
+        /// </summary>
         public IList<ILambdaFunctionSerializable> LambdaFunctions { get; } = new List<ILambdaFunctionSerializable>();
+
+        /// <summary>
+        /// Path to the CloudFormation template for the Lambda project
+        /// </summary>
         public string CloudFormationTemplatePath{ get; set; }
+
+        /// <summary>
+        /// Path to the directory containing the csproj file for the Lambda project
+        /// </summary>
         public string ProjectRootDirectory { get; set; }
+
+        /// <summary>
+        /// Whether the flag to suppress modifying the CloudFormation template 
+        /// description is set to true in the Lambda project csproj
+        /// </summary>
+        public bool IsCloudFormationDescriptionModificationSuppressed { get; set; } = false;
```

Comment:
nit: No need for the explicit assignment to `false` as it is false by default.

Reply:
Removed in amended commit.